hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -72,60 +95,54 @@ class ReadOnlyHandle
 
   double get_value() const
   {
+    // BEGIN (Handle export change): for backward compatibility
+    // TODO(Manuel) return value_ if old functionality is removed
     THROW_ON_NULLPTR(value_ptr_);
     return *value_ptr_;
+    // END
+  }
+
+  void set_value(double value)
+  {
+    // BEGIN (Handle export change): for backward compatibility
+    // TODO(Manuel) set value_ directly if old functionality is removed
+    THROW_ON_NULLPTR(this->value_ptr_);
+    *this->value_ptr_ = value;
+    // END
   }
 
 protected:
   std::string prefix_name_;
   std::string interface_name_;
+  HANDLE_DATATYPE value_;
+  // BEGIN (Handle export change): for backward compatibility
+  // TODO(Manuel) redeclare as HANDLE_DATATYPE * value_ptr_ if old functionality is removed
   double * value_ptr_;
+  // END
```

Comment:
@mamueluth what do you think about this?

Reply:
why as pointer instead of having the value directly stored in the handle?
How this should look look in the future you can see here: [handle](https://github.com/StoglRobotics-forks/ros2_control/pull/14/files#diff-76f1eebe99a92140f6b3e99d1a250e6a9cca669ecddbcb270c396b603bf16efe)
[example for system_interface](https://github.com/StoglRobotics-forks/ros2_control/pull/14/files#diff-dbaf9de12582df8a9762c4f68267da70d479584cbaddc35f9726acf3fad8302a)